### DISCUSSION TITLE
API Credits

### DISCUSSION THREAD

**Discussion by AdrianeNS** [brianpetro/obsidian-smart-connections#607] [Q&A]
I'm a little confused about the need for API credits for OpenAI.  There is quite a list of LLM models available in the plugin, and many are free to use.  For example, I'm partial to Gemini.  I'm having trouble understanding why I would need to pay for credit when the model I'm using is free.  Also, I pay for Gemini Advanced, so can I use that in the plugin?  If someone could explain this to me I'd really appreciate it!

**Comment by christianngarcia** [comment]
Currently, there is no integration with Gemini Advanced. However, you can use the Gemini models using the API. If you're looking for free models, Open Router has free API options. 🌼
